lib/exsoda/http.ex

```diff
@@ -175,7 +176,7 @@ defmodule Exsoda.Http do
     |> add_opt(user_opts, :app_token, nil)
     |> add_opt(user_opts, :recv_timeout, 5_000)
     |> add_opt(user_opts, :timeout, 5_000)
-    |> add_opt(user_opts, :params, user_opts[:params])
+    |> add_opt(user_opts, :params, Keyword.get(user_opts, :params, []))
```

Comment:
This apparently changed semantics at some point?  Anyway, making this default to `[]` rather than `nil` was required to make the tests happy.

Reply:
(I'm also not sure why it' written like this at all instead of just `add_opt(user_opts, params, [])` but I didn't feel like poking it further)